### PR TITLE
Sneeze nerf. Spread buff. Medical scanner nerf.

### PR DESCRIPTION
**Pull request by LanCartwright**
:cl: 
tweak: Sneeze now causes you lose sight and get stunned for a short amount of time
tweak: Sneeze stats have been nerfed.
tweak: Medical scanners no longer show possible cure.
tweak: Contagion spread code is changed to use amount of symptoms/2 instead of just the amount of syptoms.
/:cl:

### CODE REVIEW

**Comment by GunHog** [comment]
I dislike the negative properties of sneeze, as they remove their use for beneficial viruses. Yes, that is your intent, but I disagree with that concept.

**Comment by LanCartwright** [comment]
I've compensated for the nerf of sneeze by making it easier to get the spread you're looking for naturally.

I'll see if I can make a way to spread through sweat/touching, but no promises.

**Comment by GunHog** [comment]
Well, I guess it is okay, then..

**Comment by Archie700** [comment]
Why would you nerf the medical scanner?

Antag virologists will have a field day.

**Comment by Niknakflak** [comment]
Bad

**Comment by X-TheDark** [comment]
Advanced Medical/Viral Scanners that can be ordered from cargo/researched in RnD (when in doubt, stick it into RnD) so there isn't **only one place in the entire station that allows you to cure a virus**?

**Comment by Archie700** [comment]
I don't even see the point of lumping the medical scanner nerf with the sneeze nerf.

This is a clearly a buff to antag virologists who want to spread chaos in the station while making sure no one else can find out the cure.

**Comment by Jalleo** [comment]
Suggestion: Screen shake instead of blindness.

**Comment by GunHog** [comment]
>You can't build PANDEMIC from RnD, right?

You can build the PANDEMIC, yes. I believe there is also a spare board in tech storage, so you have two options if a traitor viro spaces his PANDEMIC after releasing EBOLA 2.0.

File: code/datums/diseases/advance/symptoms/sneeze.dm [hunk 1]
```diff
@@ -36,4 +36,6 @@ Bonus
 			else
 				M.emote("sneeze")
 				A.spread(A.holder, 5)
+				M.Stun(1)
```
Comment (X-TheDark) [inline code/datums/diseases/advance/symptoms/sneeze.dm]: Just for your information...`Stun()` still causes you to drop the item in your active hand.

Comment (LanCartwright) [inline code/datums/diseases/advance/symptoms/sneeze.dm]: I'll see if I can find a "root" option.

**Comment by optimumtact** [comment]
How is sneeze making you blind and stunned a nerf?

Update your changelog or your title.

**Comment by Archie700** [comment]
It's a nerf for helpful viruses, but a buff for deadly viruses.

**Comment by Niknakflak** [comment]
Still bad

**Comment by YakumoChen** [comment]
It would be nice in that case if Medbay started with at least one upgraded scanner, if you're going to nerf them.

**Comment by optimumtact** [comment]
I think there have been changes that make this PR no longer reflect the original.

Please make a new one so people can reobjectively judge it. Because this one just has a negative response.

**Pull request closed without merging**